### ISSUE TITLE
Ability to specify HTML inside labels

### ISSUE THREAD

**Issue by mattsgarlata** [SpiderStrategies/kalpa-tree#373]
For https://github.com/SpiderStrategies/Scoreboard/pull/10066, we need to style part of the tree label with a lighter font color than other text.  The tree will look like this

![387a2530-0285-11e7-8724-185ce02b19b7](https://cloud.githubusercontent.com/assets/1598200/23669975/23b77544-0334-11e7-8e5e-7a5544bbe280.png)

The HTML will look like this

![923cffde-0285-11e7-9c5a-79ffae597108](https://cloud.githubusercontent.com/assets/1598200/23669979/2888cf82-0334-11e7-9809-3febb3ab7ab2.png)

The trouble is, right now the tree is escaping the label, but I need to have control over that.  Didn't we run into a similar issue elsewhere?  Maybe in scoreboard-select?  I think the solution we went with was to not do escaping in the component, and instead do that outside the component.  I'm not sure how excited I am about having the tree not HTML escape by default, though.  Maybe we could pass an option into the tree constructor like `htmlLabel: true`

**Comment by nathanbowser** [comment]
We added the ability to control an entire node contents (i.e. `node-contents`) back when the tree was being used as a draggable list. So something like this works:

```
var contents = require('kalpa-tree/lib/contents')

new Tree({
...
contents: function (selection) {
          contents.apply(this, arguments)
          selection.selectAll('.label')
                   .append('span')
                     .attr('class', 'keys')
                     .text(' (Key)')
        }
})
```
But it's kind of annoying, there is probably an easier way to adjust the label. I'll experiment.

**Comment by nathanbowser** [comment]
@mattsgarlata published v2.8.0 on npm.
